tests/www/views/test_views.py

```diff
@@ -93,6 +93,24 @@ def test_plugin_endpoint_should_not_be_unauthenticated(app):
     check_content_in_response("Sign In - Airflow", resp)
 
 
+# cd = re.sub("[`_]", "", description.strip(" \n.").strip("\""))
+# cd = re.sub("<", "<a href=\"", cd)
+# cd = re.sub(">", "\">[site]</a>", cd)
```

Comment:
Is this intentionally added or just left after debugging

Reply:
accidentally left in